### ISSUE TITLE
[Infrastructure][LocalNet] Deploy 4 node localnet in remote environment

### ISSUE THREAD

**Issue by jessicadaugherty** [pokt-network/pocket#307]
## Objective

Deploy a non-configurable 4 node localnet in a remote environment to:
- Uncover difficulties associated with finding unknown unknowns related to deploying localnet to a devnet
- Provide foundation to scope out work for more configurable devnet
- Provide foundation to scope minimum RPC requirements to interact with devnet such as missing endpoints 
- Enable completion of P2P dynamic churn 

## Goals

- [ ] Deploy a 4 node local net in a remote environment

## Deliverable

- [ ] Deploy localnet to devnet
- [ ] Create blocks that send funds (via CLI)
- [ ] Build the foundations of a devnet dashboard as discussed in https://github.com/pokt-network/pocket/issues/195
- [ ] Create or update deployment documentation on localnet --> devnet 

## Non-goals / Non-deliverables

- Making localnet configurable or accessible
- Scoping devnet configurations
- Scoping RPC requirements 

## General issue deliverables

- [ ] Update the appropriate CHANGELOG
- [ ] Update any relevant READMEs (local and/or global)
- [ ] Update any relevant global documentation & references
- [ ] If applicable, update the source code tree explanation
- [ ] If applicable, add or update a state, sequence or flowchart diagram using [mermaid](https://mermaid-js.github.io/mermaid/)

## Testing Methodology

- Verify that a block was created by inspecting the logs and/or DB state

---

**Creator**: @jessicadaugherty
**Co-Owners**: @Olshansk

**Comment by Olshansk** [comment]
@okdas I updated the testing methodology for this. PTAL

**Comment by okdas** [comment]
So far the network is working (validators are climbing), and the client can send commands into the network. Added the observability stack, now going to bring the make targets and dashboards on par with existing local net.

**Comment by okdas** [comment]
Synced with @vlad-she - I'll spin up a separate cluster for this so we don't have to wait on refined RBAC on existing k8s clusters, and will consolidate later if necessary.

**Comment by Olshansk** [comment]
> Synced with @vlad-she - I'll spin up a separate cluster for this so we don't have to wait on refined RBAC on existing k8s clusters, and will consolidate later if necessary.

Sounds good.

@vlad-she When do you RBAC will be refined so the protocol team doesn't need to maintain the remote environment?

**Comment by jessicadaugherty** [comment]
@okdas updated deliverables to include

> Build the foundations of a devnet dashboard as discussed in https://github.com/pokt-network/pocket/issues/195

Thank you!

**Comment by okdas** [comment]
Oh, we actually merged the last ticket, so let's close that issue.
